doc/languages-frameworks/python.section.md

```diff
@@ -585,11 +582,6 @@ The `buildPythonPackage` mainly does four things:
   environment variable and add dependent libraries to script's `sys.path`.
 * In the `installCheck` phase, `${python.interpreter} setup.py test` is ran.
 
-As in Perl, dependencies on other Python packages can be specified in the
```

Comment:
This wasn't the best description, and the reference to perl didn't seem right here.

There's also
```
The `stdenv.mkDerivation` function accepts various parameters for describing build inputs (see "Specifying dependencies"). The following are of special
interest for Python packages
```

A bit below which is a good explanation with `buildPython*` in mind.